effekt/shared/src/main/scala/effekt/machine/Analysis.scala

```diff
@@ -24,7 +24,7 @@ def freeVariables(statement: Statement): Set[Variable] =
     case Construct(name, tag, values, rest) =>
       Set.from(values) ++ (freeVariables(rest) -- Set(name))
     case Switch(value, clauses, default: Option[Clause]) =>
-      Set(value) ++ clauses.flatMap { case (tag, branch) => freeVariables(branch) } ++ default.map(freeVariables).getOrElse(Set.empty)
+      Set(value) ++ clauses.flatMap(freeVariables) ++ default.map(freeVariables).getOrElse(Set.empty)
```

Comment:
This is just stylistic, right?
(now it just uses the `def freeVariables(taggedClause: (Int, Clause)): Set[Variable] = freeVariables(taggedClause._2)` instance, but it's exactly the same computation)

Reply:
btw, IntelliJ suggests `default.fold(Set.empty)(freeVariables)`, but I'd disregard that 😇

Reply:
Please refactor and deduplicate this with the other occurence